src/components/Calendar/Section.js

```diff
@@ -1,10 +1,12 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { withStyles } from '@material-ui/styles';
 import Paper from '@material-ui/core/Paper';
+import Modal from '@material-ui/core/Modal';
 import Typography from '@material-ui/core/Typography';
 import { getDurationInHours, getFormattedClassEvent } from 'util/time';
 
+
 export const MAX_WIDTH_PERCENT = 97;
 
 export const styles = {
```

Comment:
Can you also add styles so that the sections show the pointer cursor style on hover?